### PR TITLE
AIR-1133, upgrades and fixes for k8s 1.25

### PR DESCRIPTION
**Pull request by Rohitrajak1807**

### CODE REVIEW

File: Dockerfile
Comment (mithilarun) [inline Dockerfile:1]: Maybe keep this in the original location. Not sure if there is automation that depends on it

File: Makefile [hunk 1]
```diff
@@ -2,11 +2,11 @@ SRC_DIR=$(shell pwd)
...
-GOSRC=$(GOPATH)/src
-K8SNIFF_SRC_DIR=$(GOSRC)/github.com/kubermatic/k8sniff
-DEP_SRC=$(GOSRC)/github.com/golang/dep
-DEP_EXE=$(DEP_SRC)/cmd/dep/dep
-VENDOR_DIR=$(K8SNIFF_SRC_DIR)/vendor
+#GOSRC=$(GOPATH)/src
+#K8SNIFF_SRC_DIR=$(GOSRC)/github.com/kubermatic/k8sniff
+#DEP_SRC=$(GOSRC)/github.com/golang/dep
+#DEP_EXE=$(DEP_SRC)/cmd/dep/dep
+#VENDOR_DIR=$(K8SNIFF_SRC_DIR)/vendor
```
Comment (mithilarun) [inline Makefile]: Can we just remove these lines?

**Review by mithilarun — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Rohitrajak1807) [reply Makefile]: yeah, we need more Makefile changes

File: Dockerfile
Comment (Rohitrajak1807) [reply Dockerfile:1]: https://teamcity.platform9.horse/admin/editRunType.html?id=buildType:Pf9project_DeccoBuilds_K8sniff&runnerId=RUNNER_19&cameFromUrl=%2Fadmin%2FeditBuildRunners.html%3Fid%3DbuildType%253APf9project_DeccoBuilds_K8sniff%26init%3D1&cameFromTitle=
the automation seems out of date for new go versions, should we just change the automation?

**Pull request merged**